modules/io/src/writers/xviz-binary-writer.js

```diff
@@ -14,9 +14,30 @@
 
 import {XVIZBaseWriter} from './xviz-base-writer';
 import {GLTFBuilder} from '@loaders.gl/gltf';
-import {toBuffer} from '@loaders.gl/core';
 import {DracoWriter, DracoLoader} from '@loaders.gl/draco';
 import {_packBinaryJson as packBinaryJson} from '@xviz/builder';
+import {XVIZ_GLTF_EXTENSION} from '@xviz/parser';
```

Comment:
This seems like it should be in @xviz/io, but we can handle that later

Reply:
Yeah right now `io` imports `parser` but not the other way around so that is where I put it.